bigbluebutton-html5/imports/api/2.0/cursor/server/handlers/cursorUpdate.js

```diff
@@ -0,0 +1,15 @@
+import Logger from '/imports/startup/server/logger';
+import { check } from 'meteor/check';
+import updateCursor from '../modifiers/updateCursor';
+
+export default function handleCursorUpdate({ header, body }) {  
+  const meetingId = header.meetingId;
```

Comment:
It's a mistake to think that there's such a thing as recovering functionality from an older release. All this does is make you do the work twice instead of doing right the first time.